### PR TITLE
Graphql update

### PR DESCRIPTION
**Pull request by mmahalwy**
Update graphql types

### CODE REVIEW

**Review by tifa2UP — APPROVED** [review]

**Review by naveed-ahmad — APPROVED** [review]

**Comment by naveed-ahmad** [comment]
@mmahalwy merging this PR, there are some issues, some fields we should not expose. I'll fix them in the next PR, also changed some relations( like we don't need polymorphic relation "resource" for audio files. We've audio file only for verses, removing polymorphic will have slightly better index performance.)

File: app/controllers/graphql_controller.rb [hunk 1]
```diff
@@ -1,35 +1,50 @@
...
     operation_name = params[:operationName]
     context = {
       # Query context goes here, for example:
       # current_user: current_user,
     }
-    result = QuranAPISchema.execute(query, variables: variables, context: context, operation_name: operation_name)
+    result = QuranApiSchema.execute(query, variables: variables, context: context, operation_name: operation_name)
     render json: result
+  rescue => e
+    raise e unless Rails.env.development?
```
[the remark this answers is not in the document]
Comment (mmahalwy) [reply app/controllers/graphql_controller.rb:18]: @naveed-ahmad comes from graphql generator

**Comment by mmahalwy** [comment]
yeah agree about polymorphic for audio. although maybe we keep it if we have audio for a word?

**Comment by naveed-ahmad** [comment]
@mmahalwy words audio are saved in words table.

**Pull request merged**